packages/logger/index.ts

```diff
@@ -14,12 +14,13 @@
  * limitations under the License.
  */
 
-import { instances, LogLevel } from './src/logger';
+import { instances, LogLevel, setDefaultLogLevel } from './src/logger';
```

Comment:
Default log level should not be changed. It's meant to be a constant. To change the the log level, just change the value of the inst.logLevel.

Reply:
Yes, inst.logLevel works retroactively but fails to mute future instantiations of the logger. Under this design, changing the log-level globally from the "setLogLevel" function now requires either the implementer to design a monitoring service around the logger that changes newly-created instances back to the desired level or for each requiring class to be contracted to provide a logLevel from it's parent instantiation.

Seems like a hassle to me. Still, you're the maintainer of the code and if that's how you say it's gonna be then I'll make it happen. I'm open to suggestions on how to solve the global log level puzzle as well, but in the meantime I'll change this PR (within the next few minutes) with an alternative approach to spam the provided setLogLevel function every time the testing-sdk creates a new logger instance (which is only referenced by exported functions and not the database instance itself). It's not ideal for code quality but at least it will work.

Reply:
FWIW- It seems like with the current design / implementation, `setLogLevel()` is really `setLogLevelOfAllExistingLoggerInstances()` and `setDefaultLogLevel()` is really `setLogLevelOfAllFutureLoggerInstances()`.

I'm not sure why we'd want to make this distinction.  I think calling `setLogLevel()` should change the log level for all existing and future instances.  So I think `defaultLogLevel` should really be `currentLogLevel` and `setLogLevel()` should go ahead and update it in addition to notifying all existing instances of the new log level.

Reply:
That's basically what I originally did! :1st_place_medal: I just didn't rename defaultLogLevel to currentLogLevel. I've made changes with an alterante method. I would prefer doing what you suggest. Lets see if @Feiyang1  has any thoughts!